docs/scan.md

```diff
@@ -0,0 +1,86 @@
+# Scanning
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+-   [How It Works](#how-it-works)
+    -   [Pre-commit Hook](#pre-commit-hook)
+    -   [Baseline File](#baseline-file)
+        -   [Notable Fields](#notable-fields)
+-   [What Gets Scanned?](#what-gets-scanned)
+    -   [Secret Verification](#secret-verification)
+-   [How It’s Used](#how-its-used)
+-   [Excluding Files](#excluding-files)
+-   [Plugins](#plugins)
+-   [Adjusting the Scan Sensitivity](#adjusting-the-scan-sensitivity)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## How It Works
+
+`detect-secrets scan` scans the entire codebase and outputs a snapshot of currently identified secrets.
+
+This snapshot should be stored in a baseline file and updated on an as-needed basis. The `detect-secrets-hook` - i.e. pre-commit hook - will notify you when your baseline file needs to be updated.
+
+### Pre-commit Hook
+
+The pre-commit hook uses `detect-secrets`'s scanning functionality to scan your code before it gets committed. It's recommended to set up this hook ([docs](./docs/../developer-tool-faq.md#how-do-i-set-up-the-pre-commit-hook)) to prevent leaks before they reach GitHub.
+
+It's also known as the [`detect-secrets-hook`](./detect-secrets/pre_commit_hook.py).
+
+### Baseline File
+
+This file contains the output of a scan. This includes a list of detected secrets, plugins used during scanning and their settings, and line & file exclusion info. After the baseline file has been created or updated, it's a good idea to [audit](./audit.md) it. For simplicity's sake, we'll focus on scanning in this document.
+
+#### Notable Fields
+
+You'll find a **`results`** object, which contains a list of file paths corresponding to detected token data; for example:
+
+```json
+  "results": {
+    "detect_secrets/plugins/private_key.py": [
+      {
+        "hashed_secret": "daefe0b4345a654580dcad25c7c11ff4c944a8c0",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 45,
+        "type": "Private Key",
+        "verified_result": null
+      },
+```
```

Comment:
I'm not sure why this is being flagged as an error. I don't see any issues here based off the [rendered markdown](https://github.com/IBM/detect-secrets/blob/6aa429dd1f8eca12cb1733dc14a52c427f461f32/docs/scan.md).